### PR TITLE
Initial SameSite FAT Suite checkin

### LINKED ISSUE #10714: Initial test suite for SameSite
Create an initial set of tests to test current functionality and add some additional tests for future functionality. We can add to these  tests as we develop a solution.

### PR DESCRIPTION
**Pull request by pnicolucci**
fixes #10714

### CODE REVIEW

**Comment by pnicolucci** [comment]
#run-libby-bot

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#run-libby-bot

File: dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCSameSiteCookieAttributeTests.java [hunk 1]
```diff
@@ -0,0 +1,1338 @@
...
+     * Cookie cookieOne = new Cookie("cookieOne", "cookieOne");
+     *
+     * Configure the server to mark this Cookie as SameSite=Lax and verify.
+     *
+     * Verify that the Set-Cookie header is correct.
+     *
+     *
+     * @throws Exception
+     */
+    //@Test
```
Comment (mrsaldana) [inline dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCSameSiteCookieAttributeTests.java]: This test is still commented, should it be?

**Pull request merged**